<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\dev\dynamo\Dynamo.Tests\bin\Debug\Dynamo.Tests.dll" total="6" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2013-03-30" time="23:49:29">
  <environment nunit-version="2.6.2.12296" clr-version="2.0.50727.5466" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\dev\dynamo" machine-name="FEZZRIG" user="my" user-domain="FEZZRIG" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\dev\dynamo\Dynamo.Tests\bin\Debug\Dynamo.Tests.dll" executed="True" result="Failure" success="False" time="3.834" asserts="0">
    <results>
      <test-suite type="Namespace" name="Dynamo" executed="True" result="Failure" success="False" time="3.813" asserts="0">
        <results>
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="3.813" asserts="0">
            <results>
              <test-suite type="TestFixture" name="DynamoTests" executed="True" result="Success" success="True" time="3.346" asserts="0">
                <results>
                  <test-case name="Dynamo.Tests.DynamoTests.Can_create_dynamo_instance" executed="True" result="Success" success="True" time="0.821" asserts="1" />
                  <test-case name="Dynamo.Tests.DynamoTests.Can_create_ruby_instance" executed="True" result="Success" success="True" time="2.473" asserts="1" />
                  <test-case name="Dynamo.Tests.DynamoTests.Can_query_bridged_types" executed="True" result="Success" success="True" time="0.028" asserts="1" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="ImpromptuTests" executed="True" result="Failure" success="False" time="0.457" asserts="0">
                <results>
                  <test-case name="Dynamo.Tests.ImpromptuTests.Should" executed="True" result="Success" success="True" time="0.341" asserts="1" />
                  <test-case name="Dynamo.Tests.ImpromptuTests.Should_be_able_to_impromptu_class_instance" executed="True" result="Error" success="False" time="0.013" asserts="0">
                    <failure>
                      <message><![CDATA[Microsoft.CSharp.RuntimeBinder.RuntimeBinderException : 'Dynamo.Tests.Product' does not contain a definition for 'Buy']]></message>
                      <stack-trace><![CDATA[at CallSite.Target(Closure , CallSite , Object , Func`3 )
at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
at Dynamo.Tests.ImpromptuTests.Should_be_able_to_impromptu_class_instance() in c:\dev\dynamo\Dynamo.Tests\ImpromptuTests.cs:line 46
]]></stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="Dynamo.Tests.ImpromptuTests.Should_create_dynamic_factory" executed="True" result="Success" success="True" time="0.097" asserts="1" />
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>